projects/js-packages/charts/src/components/shared/with-responsive.tsx

```diff
@@ -32,14 +32,13 @@ export function withResponsive< T extends BaseChartProps< unknown > >(
 
 		// Calculate dimensions
 		const containerWidth = parentWidth ? Math.min( parentWidth, maxWidth ) : 600;
-		const containerHeight = containerWidth * aspectRatio;
+		const containerHeight = props.height ?? containerWidth * aspectRatio;
```

Comment:
Users should still be able to specify the height so that it'll be a different ratio even when responsive.